metals/src/main/scala/scala/meta/internal/tvp/MetalsTreeViewProvider.scala

```diff
@@ -67,7 +68,7 @@ class MetalsTreeViewProvider(
       )
     },
     { (id, symbol) =>
-      doCompile(id)
+      if (isBloop()) doCompile(id)
```

Comment:
Should be good for a quick fix for sure. I haven't been able to take a proper look here, but I think this is perfectly fine.